CHANGELOG.md

```diff
@@ -50,25 +52,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - $_WD_JSON\_* constants
-- Checks for valid installation directory (_WD_UpdateDriver)
-- Check for existing 32/64 bit driver (_WD_UpdateDriver)
+- _WD_UpdateDriver: Checks for valid installation directory
+- _WD_UpdateDriver: Check for existing 32/64 bit driver
 
 ### Changed
 
-- Sets @Extended to shown context changed (_WD_GetElementFromPoint)
-- Additional error checking (_WD_GetElementFromPoint)
+- _WD_GetElementFromPoint: Sets @Extended to shown context changed
+- _WD_GetElementFromPoint: Additional error checking
 - _WD_ExecuteScript: Add support for return of additional subnodes
 - Update various functions to use $_WD_JSON\_* constants
```

Comment:
redundant backslash ?

Reply:
Huh? You mean this? `$_WD_JSON\_* constants`

If so, I believe that @Sven-Seyfert added that in a17efcd7ae86ed63383275c0b1877049f4ab4d67.

Reply:
But why ? I do not understand it.
@Sven-Seyfert could you elaborate on this ?